.htmllintrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I wish there were similar projects as `standard` so that we don't need to create two extra metadata files.